src/common/NetworkEventManager.ts

```diff
@@ -1,7 +1,7 @@
 import { Protocol } from 'devtools-protocol';
 import { HTTPRequest } from './HTTPRequest.js';
 
-export type QueuedEvents = {
+export type QueuedEventGroup = {
   responseReceived: Protocol.Network.ResponseReceivedEvent;
```

Comment:
Naming idea: `responseReceivedEvent`, `loadingFinishedEvent`, and `loadingFailedEvent`. Call the whole thing a `QueuedResponseEventGroup`